options.js

```diff
@@ -25,14 +28,15 @@ function restore_options() {
     chrome.storage.sync.get({
         github_access_token: '',
         hide_labels_on_closed_prs: true,
-        hide_closed_prs: false
+        hide_closed_prs: false,
+        pr_columns: '',
```

Comment:
We're setting a default for pr_columns in content.js, but there is only placeholder text here.  So if they wanted to remove one of the defaults, they'd have to retype the whole thing.  Let's prefill this line with `In Review, Github Review` -- other than that, this PR looks great!